src/pyledger.cc

```diff
@@ -43,6 +43,23 @@ BOOST_PYTHON_MODULE(ledger)
 {
   using namespace ledger;
 
+  scope().attr("__author__") = "John Wiegley <jwiegley@gmail.com>";
+  scope().attr("__version__") = Ledger_VERSION;
+  scope().attr("__date__") = Ledger_VERSION_DATE;
+  scope().attr("__doc__") =
+    "Python API Documentation\n\n"
+    "Documentation of the Ledger Python API is an ongoing process and you are invited\n"
+    "to help out and contribute. In case you find this documentation incorrect,\n"
+    "incomplete, unclear, or lacking please open a pull request at\n"
+    "https://git.ledger-cli.org/ledger/pulls."
+    ;
+
+#if !DEBUG_MODE
+  docstring_options doc_options;
+  doc_options.disable_cpp_signatures();
```

Comment:
For details about docstring_options see https://www.boost.org/doc/libs/1_82_0/libs/python/doc/html/reference/high_level_components/boost_python_docstring_options_h.html